SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -28,7 +28,7 @@ struct FConnectionConfig
         
 #if PLATFORM_IOS || PLATFORM_ANDROID
 		// On a mobile platform, you can only be a client worker, and therefore use the external IP.
-		WorkerType = SpatialConstants::ClientWorkerType;
+		WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
```

Comment:
There's some discussion around this going on in https://improbableio.atlassian.net/browse/UNR-1763